File: src/main/java/cn/hirrao/javaee/entity/Article.java
package cn.hirrao.javaee.entity;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import org.springframework.data.annotation.Id;


@Getter
@Setter
@AllArgsConstructor
public class Article {
    @Id
    private Long id;

    private String title;
    private String description;
    private String image;
    private String content;

    // getters and setters
}

File: api/src/main/kotlin/cn/hirrao/javaee/entity/Article.kt
package cn.hirrao.javaee.entity

import com.baomidou.mybatisplus.annotation.TableName
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter
import org.springframework.data.annotation.Id


@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@TableName("articles")
class Article {
    @Id
    var id: Long? = null

    var title: String? = null
    var description: String? = null
    var image: String? = null
    var content: String? = null // getters and setters
}
